packages/ketchup/src/components/kup-activity-timeline/kup-activity-timeline-declarations.ts

```diff
@@ -0,0 +1,42 @@
+import { KupDataColumn, KupEventPayload } from '../../components';
+import { KupDataRow } from '../../managers/kup-data/kup-data-declarations';
+
+export enum KupActivityTimelineProps {
+    data = 'Dataset containing the activities list',
+    dateColumn = 'Column containing dates',
+    timeColumn = 'Column containing times',
+}
+
+export interface KupActivityTimelineData {
+    title: string;
+    value: string;
+    columnName: string;
+    cellId: string;
+}
+
+export interface KupActivity {
```

Comment:
Prefixes for components' declarations should be structured like this: KupComponentName.
Some declarations must be fixed in this regard.